less/pages/encrypt/components.less

```diff
@@ -177,6 +177,7 @@ video{
   position: relative;
   margin: 5px;
   flex: 1 1 40%;
+  min-width: 0;
```

Comment:
Could just be as simple as `max-width: 100%;` doesn't do expected thing in firefox.